src/main/java/hudson/remoting/jnlp/Main.java

```diff
@@ -246,12 +249,12 @@ public static void main(String[] args) throws IOException, InterruptedException
      * Main without the argument handling.
      */
     public static void _main(String[] args) throws IOException, InterruptedException, CmdLineException {
-        // TODO skip this on Java 17+ (e.g. io.jenkins.lib.versionnumber.JavaSpecificationVersion) as it prints a warning
-        // otherwise needed for JavaWebStart agents (JENKINS-67000)
```

Comment:
I think you want to keep some comments explaining the context, specifically https://github.com/jenkinsci/remoting/pull/507#issuecomment-1030341067

Reply:
Indeed, added.